<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XexDialog</class>
 <widget class="QDialog" name="XexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XEX Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Images/Leaner24.png</normaloff>:/Images/Leaner24.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>System Import Libraries</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lstSystemImportLibraries"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Module Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkTitleModule">
        <property name="text">
         <string>Title Module</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkExportsToTitle">
        <property name="text">
         <string>Exports to Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSystemDebugger">
        <property name="text">
         <string>System Debugger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDLL">
        <property name="text">
         <string>DLL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPatch">
        <property name="text">
         <string>Patch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkFullPatch">
        <property name="text">
         <string>Full Patch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDeltaPatch">
        <property name="text">
         <string>Delta Patch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUserMode">
        <property name="text">
         <string>User Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Ratings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="imgEsrbRating">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>70</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="imgPegiRating">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>66</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>66</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
